src/Components/Notifications/NotificationsPills.tsx

```diff
@@ -0,0 +1,48 @@
+import { ActionType } from "@artsy/cohesion"
+import { Flex, Pill } from "@artsy/palette"
+import { NotificationType } from "Components/Notifications/types"
+import { useNotificationsContext } from "Components/Notifications/useNotificationsContext"
+import { useTracking } from "react-tracking"
+
+export const NOTIFICATIONS_PILLS = [
+  { value: "All", name: "all" },
+  { value: "Alerts", name: "alerts" },
+  { value: "Following", name: "following" },
+]
+export const NotificationsPills: React.FC = () => {
+  const { trackEvent } = useTracking()
+  const { setCurrentNotificationFilterType, state } = useNotificationsContext()
+
+  const handleClick = tabNumber => {
+    setCurrentNotificationFilterType(
+      NOTIFICATIONS_PILLS[tabNumber].name as NotificationType
+    )
+  }
+
+  const sendAnalytics = pill => {
```

Comment:
**minor:** the attribute named `pill` is a little misleading as we already have a value of a different type with the same name. What do you think about renaming it? `pillName` for example?